oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreServiceConfigurationTest.java

```diff
@@ -239,6 +239,17 @@ public void overrideFullGCExcludePaths() throws Exception {
         assertArrayEquals(new String[]{"/foo", "/bar"}, config.fullGCExcludePaths());
     }
 
+    @Test
+    public void perfLoggerInfoMillisTest() throws Exception {
+        addConfigurationEntry(preset, "perfLoggerInfoMillis", 0L);
```

Comment:
I would suggest to add a test for the default (which as mentioned above should not be zero in my view)

Reply:
Done.